docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc

```diff
@@ -28,41 +28,41 @@ Ensure you have:
 . Once the _WFM Project_ is created, click *Finish*
 .. You are brought to the _WFM Project_ homepage
 
-[id='{context}-setting-up-and-deploying-the-wfm-demo-cloud-app']
+[id='{context}-setting-up-and-deploying-the-wfm-cloud-app']
 [discrete]
-== Setting Up and Deploying the WFM Demo Cloud App
+== Setting Up and Deploying the {RainCatcher} Cloud App
 
-. On the _WFM Project_ homepage, click on the *WFM Demo Cloud App*
+. On the _WFM Project_ homepage, click on the *WFM Cloud App*
 . In the left side navigation panel, click on *Data Browser*
 . If the database requires an update, select *Upgrade database*
 .. A warning message appears, click on *Upgrade Now* to proceed
 . Once the upgrade is complete, click *Next*
 . Navigate to the deployment screen by clicking *Deploy*
-. On the deployment screen, select *Node.js - 6.9.1* from the *Choose a Runtime* list
-. To deploy the _WFM Demo Cloud App_, click on the *Deploy Cloud App*
+. On the deployment screen, select *{WFM-RC-NodeVersion}* from the *Choose a Runtime* list
```

Comment:
@austincunningham , @wtrocki - variable now added - current value is : 

:WFM-RC-NodeVersion: 6 LTS